wallet/udb/addressmanager.go

```diff
@@ -2152,14 +2156,14 @@ func ValidateAccountName(name string) error {
 // it requires the manager to be unlocked.
 func (m *Manager) NewAccount(ns walletdb.ReadWriteBucket, name string) (uint32, error) {
 	if m.watchingOnly {
-		return 0, managerError(ErrWatchingOnly, errWatchingOnly, nil)
+		return 0, managerError(apperrors.ErrWatchingOnly, errWatchingOnly, nil)
 	}
 
 	m.mtx.Lock()
 	defer m.mtx.Unlock()
```

Comment:
fetchAccountByName only looks in the db bucket itself and does not touch any of the manager's cached data.